js/src/OnBoarding.js

```diff
@@ -432,23 +432,25 @@ class OnBoarding
    */
   setShutDown(value)
   {
-    this.save({action: 'setShutDown', value: value ? 1 : 0}, ((error) => {
-      this.isShutDown = value;
+    this.isShutDown = value ? 1 : 0;
+
+    if (1 === this.isShutDown) {
+      $('.onboarding-advancement').toggle(false);
+      $('.onboarding-navbar').toggleClass('displayed', true);
+      $('.onboarding-popup').remove();
+      $('.onboarding-tooltip').remove();
+    }
+
+    this.save({action: 'setShutDown', value: this.isShutDown}, ((error) => {
```

Comment:
alright, but then with this new code we don't have to wait when we close the popin
but if the user wants to see the next step he still has to wait for the save request to return
is it what we want? it might be, maybe we need to be sure everything has been displayed / loaded correctly before we show the next step

Reply:
Indeed, when we want to bring back the on boarding process, we wait for the server response before doing anything.

Only the shutdown imediately do something on the page.

Reply:
Ok perfect then!